ovirtlago/virt.py

```diff
@@ -50,7 +51,7 @@ def _create_vm(self, vm_spec):
         else:
             provider_name = vm_spec.get(
                 'vm-type',
-                lago.config.get('default_vm_provider', 'default'),
+                lago_config.get('default_vm_provider', 'default'),
```

Comment:
With the old `lago.config` it was not masked, as the other is namespaced behind the `lago` module, but :+1: for the initiative (no need to revert this, it's ok as it is too).